src/main/java/com/example/onlinebookstore/OnlineBookStoreApplication.java

```diff
@@ -1,13 +1,43 @@
 package com.example.onlinebookstore;
 
+import com.example.onlinebookstore.model.Book;
+import com.example.onlinebookstore.service.BookService;
+import java.math.BigDecimal;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class OnlineBookStoreApplication {
+    private final BookService bookService;
```

Comment:
you can pass `BookService bookService` as a parameter in commandLineRunner() and remove this field